elytron/src/test/java/org/wildfly/extension/elytron/LdapTestCase.java

```diff
@@ -146,12 +147,14 @@ private static void loadResources() throws Exception {
     public static void startLdapService() throws Exception {
         loadResources();
         TestEnvironment.startLdapService();
+        Security.setProperty("keystore.PKCS12.keyProtectionAlgorithm", "PBEWithHmacSHA256AndAES_128");
```

Comment:
It would be good to save the old value that gets returned here and then make use of that value when cleaning up (e.g., see KeyStoresTestCase).

Reply:
Fixed!